cli/flox/src/utils/search.rs

```diff
@@ -190,6 +191,8 @@ pub struct DisplaySearchResults {
     count: Option<u64>,
     /// number of actual results (including duplicates)
     n_results: u64,
+    /// Whether to bold the search term matches in the output
+    bold_search_term: bool,
```

Comment:
Suggestion: I would call this something like `use_bold`. Right now we would have `search_term` and `bold_search_term`, and `bold_search_term` sounds like the bolded version of `search_term`.